python/cuBERT/pybind/cuBERT_pybind.cc

```diff
@@ -0,0 +1,314 @@
+/**
+ * Copyright (c) 2019-present, NTC, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/numpy.h>
+#include <vector>
+
+#include <cuBERT.h>
+#include <half.hpp>
+
+
+namespace py = pybind11;
+
+
+char *int2char(const std::vector<int> input) {
+  unsigned int sz = input.size();
+
+  char *result = new char[sz + 1];
+  result[sz] = '\0';
+
+  for (unsigned int i = 0; i < sz; ++i) {
+    result[i] = static_cast<char>(input[i]);
+  }
+
+  return result;
+}
+
+char **str2char(const std::vector<std::string> input) {
+  unsigned int sz = input.size();
+
+  char **result = new char *[sz + 1];
+  result[sz] = nullptr;
+
+  for (unsigned int i = 0; i < sz; ++i) {
+    unsigned int len = input[i].size();
+    result[i] = new char[len + 1];
+    std::strcpy(result[i], input[i].c_str());
```

Comment:
I think we do not need to copy these strings. It is OK just pointing to them (if they are also `\0` terminated).

Reply:
copy is needed. it's not enough for just pointing to them(Tested)